pages/linux/turbostat.md

```diff
@@ -0,0 +1,24 @@
+# turbostat
+
+> Report processor frequency and idle statistics.
```

Comment:
```suggestion
> Report processor topology, frequency, temperature, power, and idle statistics.
```
I took this description from the "DESCRIPTION" section of the man page, this probably describes all information `turbostat` can provide.